app/models/organization.js

```diff
@@ -11,4 +11,5 @@ export default Owner.extend({
   iconLargeUrl: attr(),
   projects: DS.hasMany('project', { async: true }),
   members: DS.hasMany('user', { async: true }),
+  organizationMemberships: DS.hasMany('organization-membership', { async: true }),
```

Comment:
We have to explicitly do this on _all_ relationships like this (where we need extra information on the join) or weirdness will arise.